DQMServices/FileIO/plugins/DQMFileSaverPB.cc

```diff
@@ -43,6 +43,13 @@ DQMFileSaverPB::DQMFileSaverPB(const edm::ParameterSet& ps) : DQMFileSaverBase(p
   if (tag_ != "UNKNOWN") {
     streamLabel_ = "DQMLive";
   }
+
+  if (!fakeFilterUnitMode_) {
+    evf::EvFDaqDirector* daqDirector = (evf::EvFDaqDirector*)(edm::Service<evf::EvFDaqDirector>().operator->());
+    const std::string initFileName = daqDirector->getInitFilePath(streamLabel_);
+    std::ofstream file(initFileName);
+    file.close();
```

Comment:
I wondering if one should check the validity of the output file, and issue a warning/error/exception if the check fails; this check is done for the OutputModule, but not for the other streams.

Reply:
Here we now throw exception if there is any problem with the file or DaqDirector is not available (the latter can be required since it's also used elsewhere in the module).